tests/playbooks/centos7_deps.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install CentOS 7 deps for nmstate tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+      - git
+      - python2-pip
+      - NetworkManager
```

Comment:
NetworkManager is not installed by default?

Reply:
It is not in cloud images.